index.html

```diff
@@ -0,0 +1,325 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+    <body>
+      <!--work section-->  
+      <header class="navbar">
+        <ul>
+          <li class="navbar-logo">Tadesse J.</li>
+          <li class="navbar-toggle"><i class="fa fa-bars"></i></li>
+          <li class="navbar-links"><a href="#"></a></li>
+          <li class="navbar-links"><a href="#"></a></li>
+          <li class="navbar-links"><a href="#"></a></li>
+        </ul>
+       </header>
+        <!-- HOME SECTION -->
+        <section class="container-fluid">
+            <div class="home-">
+              <h1 class="heading-home">
+                Hey There.
+                I’m Tadesse
+              </h1>
+              <h2 class="card-title">I’m a software developer</h2>
+              <p class="card-text">
+                I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and have a
+                project you need coded, don’t hestiate to contact me.
+              </p>
+            </div>
+          <div>
+            <ul class="element">
+              <li>
+                <a href="#">
+                  <i class="bi bi-twitter text-white"></i>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <i class="bi bi-linkedin text-white"></i>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <i class="bi bi-instagram text-white"></i>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <i class="bi bi-github text-white"></i>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </section>
+        <section class="work">
+            <div class="row">
+            <div class="recent-work">
+                <h1 class="work-heading">My Recent Works</h1>
```

Comment:
- [ ]  I suggest you use `display: grid` on the container to layout the content of the works fine on a mobile phone. Using `grid` for styling makes your website flexible and responsive.